interface/forms/newpatient/common.php

```diff
@@ -411,8 +411,8 @@ function cancelClickedOld() {
      <td class='bold' nowrap><?php echo xlt('Date of Service:'); ?></td>
      <td class='text' nowrap>
       <input type='text' size='10' class='datepicker input-sm' name='form_date' id='form_date' <?php echo $disabled ?>
-       value='<?php echo $viewmode ? substr($result['date'], 0, 10) : date('Y-m-d'); ?>'
-       title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
+       value='<?php echo $viewmode ? oeFormatShortDate(substr($result['date'], 0, 10)) : oeFormatShortDate(date('Y-m-d')); ?>'
```

Comment:
need to wrap both oeFormatShortDate() with attr() function to prevent xss

Reply:
Done

Reply:
still missing the attr() here

Reply:
TODO:
I will add the attr()
:)